content/en/blog/_posts/0000-00-00-kube-proxy-non-privileged.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: blog
+title: "Kubernetes 1.29: Support for Running kube-proxy in a Non-privileged Container"
+date: 0000-00-00
+slug: kube-proxy-non-privileged
+---
+
+**Author**: Lars Ekman
+
+This post describes how the `--init-only` flag to `kube-proxy`, added in K8s v1.29, can
+be used to perform configuration that requires privileged mode in an
+initContainer, while the main `kube-proxy` container may run with a
+stricter securityContext.
+
+Please note that `kube-proxy` can be installed in different ways. For
+installers such as `kubeadm` that install `kube-proxy` to run as a Pod, the new
+functionality can help you improve security.
```

Comment:
This can help you improve security if it _doesn't_ run as a pod too...

```suggestion
Please note that `kube-proxy` can be installed in different ways. The
examples below assume that kube-proxy is run from a pod, but similar
changes could be made in clusters where it is run as a system service.
```